src/main/kotlin/com/autonomousapps/model/intermediates/producers.kt

```diff
@@ -20,6 +21,7 @@ internal interface DependencyView<T> : Comparable<T> where T : DependencyView<T>
  * nb: Deliberately does not implement [DependencyView]. For various reasons, this information gets embedded in
  * [ExplodedJar], which is the preferred access point for deeper analysis.
  */
+@JsonClass(generateAdapter = true)
```

Comment:
This class is also used in serialization but wasn't annotated before, not sure if there's others but they'll need to be updated with this too unless you want to just defensively keep the reflection adapter

Reply:
Just in general, why do some additional data classes and enum classes now need this annotation?

Reply:
since it's an annotation processor, it needs an annotation to process to know what to generate for. It also ends up being valuable for seeing at-a-glance which classes are relevant for JSON parsing, including enums. It's particularly useful in R8/proguard situations too as there are embedded rules to keep these.